python/ray/data/tests/test_operators.py

```diff
@@ -104,11 +114,14 @@ def test_map_operator_bulk(ray_start_regular_shared, use_actors):
     while input_op.has_next():
         op.add_input(input_op.get_next(), 0)
     op.inputs_done()
-    for work in op.get_work_refs():
-        ray.get(work)
-        op.notify_work_completed(work)
+    work_refs = op.get_work_refs()
+    while work_refs:
+        for work_ref in work_refs:
+            ray.get(work_ref)
+            op.notify_work_completed(work_ref)
+        work_refs = op.get_work_refs()
     if use_actors:
-        assert op.progress_str() == "0 actors"
+        assert op.progress_str() == "1/1 actors"
```

Comment:
I think it may be more informative to show "<num_ready> (<num_pending> pending)" like the legacy backend does instead. I'm not sure the denominator is useful given it's from the config.

Reply:
Agreed, I'll make that change!